File: url/src/test/java/com/ruimaranhao/URLValidatorTest.java
package com.ruimaranhao;

import org.junit.Test;

import java.io.IOException;

import static junit.framework.Assert.assertFalse;
import static junit.framework.Assert.assertTrue;

public class URLValidatorTest {

    @Test
    public void validURL() throws IOException {
        assertTrue(URLValidtor.isURL("abc.com"));

    }

    @Test
    public void invalidURL() throws IOException {
        assertFalse(URLValidtor.isURL("what"));

    }
}

File: url/src/test/kotlin/com/maranhao/URLValidatorTest.kt
package com.maranhao


import com.ruimaranhao.URLValidtor
import org.junit.Test
import java.io.IOException
import kotlin.test.assertFalse
import kotlin.test.assertTrue

class URLValidatorTest {

    @Test
    @Throws(IOException::class)
    fun validURL() {
        assertTrue(URLValidtor.isURL("abc.com"))

    }

    @Test
    @Throws(IOException::class)
    fun invalidURL() {
        assertFalse(URLValidtor.isURL("what"))

    }
}
